server/src/models/services/models.js

```diff
@@ -148,120 +148,68 @@ export default function (model_singleton) {
     ...bilingual_str("service_name"), // only for dept, program
     website_visits_count: { type: Number },
   });
+  const GovServiceSummarySchema = mongoose.Schema({
+    id: pkey_type(),
+    service_general_stats: ServiceGeneralStatsSchema,
+    service_type_summary: [ServiceTypeSummarySchema],
+    service_digital_status_summary: [ServiceDigitalStatusSummarySchema],
+    service_id_methods_summary: [ServiceIdMethodsSummarySchema],
+    service_standards_summary: [ServiceStandardsSummarySchema],
+    service_fees_summary: [ServiceFeesSummarySchema],
+    service_high_volume_summary: [ServicesHighVolumeSummarySchema],
+    top_services_website_visits_summary: [
+      TopServicesWebsiteVisitsSummarySchema,
+    ],
+  });
+  const OrgServiceSummarySchema = mongoose.Schema({
+    id: pkey_type(),
+    service_general_stats: ServiceGeneralStatsSchema,
+    service_type_summary: [ServiceTypeSummarySchema],
+    service_digital_status_summary: [ServiceDigitalStatusSummarySchema],
+    service_id_methods_summary: [ServiceIdMethodsSummarySchema],
+    top_services_application_vol_summary: [
+      TopServicesApplicationVolSummarySchema,
+    ],
+    top_services_website_visits_summary: [
+      TopServicesWebsiteVisitsSummarySchema,
+    ],
+    service_fees_summary: [ServiceFeesSummarySchema],
+    service_standards_summary: [ServiceStandardsSummarySchema],
+  });
+  const ProgramServiceSummarySchema = mongoose.Schema({
```

Comment:
Could simply combine org and program as they're equivalent but just leaving it separately as dept and program may diverge in the future

Reply:
Seems good to me. Maybe just to make it clearer at a glance what's common between these three schemas, as well as between the org and program schemas in particular, ou could define the common fragments above and just spread them in here?